tools/opt/opt.cpp

```diff
@@ -197,17 +195,23 @@ Options (in lexicographical order):
                'spirv-opt --merge-blocks -O ...' applies the transformation
                --merge-blocks followed by all the transformations implied by
                -O.
-  --relax-store-struct
+  --relax-struct-store
```

Comment:
I'd rather not change flag names here.  There may be external users relying on the previous name.  Better leave it for another PR.

Reply:
This option exists in the validator as --relax-struct-store, but in opt it was --relax-store-struct. This simply unifies those options since they do the same thing.